miniwin/src/d3drm/backends/software/renderer.cpp

```diff
@@ -710,7 +710,7 @@ void Direct3DRMSoftwareRenderer::SubmitDraw(
 
 	// Pre-transform all vertex positions and normals
 	m_transformedVerts.clear();
-	m_transformedVerts.reserve(mesh.vertices.size());
+	m_transformedVerts.resize(mesh.vertices.size());
```

Comment:
this is faster since resize() will cause it to init all elements.
```
m_transformedVerts.reserve(mesh.vertices.size());
for (const auto& src : mesh.vertices) {
	D3DRMVERTEX dst;
	dst.position = TransformPoint(src.position, modelViewMatrix);
	dst.normal = src.normal;
	dst.texCoord = src.texCoord;
	m_transformedVerts.push_back(dst);
}
```